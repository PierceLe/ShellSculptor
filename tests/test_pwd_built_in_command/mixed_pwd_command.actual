>> cd ~/test_data/symbolic_link_detect
cd: no such file or directory: /Users/piercele/test_data/symbolic_link_detect
>> pwd
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests
>> pwd -P
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests
>> pwd -fP
pwd: invalid option: -f
>> pwd -f
pwd: invalid option: -f
>> pwd -PPPPPPPPPPPPPPPP
pwd: invalid option: -
>> pwd -P -P -P -P -P -P
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests

