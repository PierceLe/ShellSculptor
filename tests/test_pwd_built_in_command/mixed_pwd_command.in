cd ~/test_data/symbolic_link_detect
pwd
pwd -P
pwd -fP
pwd -f
pwd -PPPPPPPPPPPPPPPP
pwd -P -P -P -P -P -P
