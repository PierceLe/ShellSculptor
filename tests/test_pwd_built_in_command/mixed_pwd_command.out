>> cd ~/test_data/symbolic_link_detect
>> pwd
/home/test_data/symbolic_link_detect
>> pwd -P
/home/test_data/PWD_TEST/Hello/How/Are/You
>> pwd -fP
pwd: invalid option: -f
>> pwd -f
pwd: invalid option: -f
>> pwd -PPPPPPPPPPPPPPPP
/home/test_data/PWD_TEST/Hello/How/Are/You
>> pwd -P -P -P -P -P -P
/home/test_data/PWD_TEST/Hello/How/Are/You

